packages/frontend-api/src/Model/Resolver/Products/ProductResolverMap.php

```diff
@@ -0,0 +1,67 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Shopsys\FrontendApiBundle\Model\Resolver\Products;
+
+use Overblog\GraphQLBundle\Resolver\ResolverMap;
+use Shopsys\FrameworkBundle\Component\Domain\Domain;
+use Shopsys\FrameworkBundle\Model\Product\Collection\ProductCollectionFacade;
+use Shopsys\FrameworkBundle\Model\Product\Product;
+
+class ProductResolverMap extends ResolverMap
+{
+    /**
+     * @var \Shopsys\FrameworkBundle\Component\Domain\Domain
+     */
+    protected $domain;
+
+    /**
+     * @var \Shopsys\FrameworkBundle\Model\Product\Collection\ProductCollectionFacade
+     */
+    protected $productCollectionFacade;
+
+    /**
+     * @param \Shopsys\FrameworkBundle\Component\Domain\Domain $domain
+     * @param \Shopsys\FrameworkBundle\Model\Product\Collection\ProductCollectionFacade $productCollectionFacade
+     */
+    public function __construct(Domain $domain, ProductCollectionFacade $productCollectionFacade)
+    {
+        $this->domain = $domain;
+        $this->productCollectionFacade = $productCollectionFacade;
+    }
+
+    protected function map()
```

Comment:
```suggestion
    protected function map(): array
```

and missing annotation

Reply:
Hmm, why targets standards-fix-diff and  standards-diff did not work?